benchexec/tools/template.py

```diff
@@ -140,6 +140,7 @@ def _version_from_tool(
         use_stderr=False,
         ignore_stderr=False,
         line_prefix=None,
+        expected_exitcode=0,
```

Comment:
In principle, I agree with this addition. I would, however, suggest that we make it a [keyword-only argument](https://peps.python.org/pep-3102/). We would never want someone to pass a single int without explicitly naming its purpose here. (The same would be true for the other parameters, but we cannot change it for them.)

Reply:
Done in 826ef032c3b6daf79df4f9800b8cd60392389051